docs/man/docker-create.1.md

```diff
@@ -56,6 +57,9 @@ IMAGE [COMMAND] [ARG...]
 **--add-host**=[]
    Add a custom host-to-IP mapping (host:ip)
 
+**--blkio-weight**=0
+   Block IO weight (relative weight), range of weights is from 10 to 1000.
```

Comment:
Update this phrase where it appears, please.

--->
Block IO weight (relative weight) accepts a weight value between 10 to 1000.